packages/lesswrong/lib/collections/userTagRels/collection.ts

```diff
@@ -0,0 +1,58 @@
+import { getDefaultMutations, getDefaultResolvers, schemaDefaultValue } from "../../collectionUtils";
+import { foreignKeyField } from "../../utils/schemaUtils";
+import { createCollection } from '../../vulcan-lib';
+
+const schema: SchemaType<DbUserTagRel> = {
+  tagId: {
+    ...foreignKeyField({
+      idFieldName: "tagId",
+      resolverName: "tag",
+      collectionName: "Tags",
+      type: "Tag",
+    }),
+    canRead: ['guests'],
+    canCreate: ['members'],
+  },
+  userId: {
+    ...foreignKeyField({
+      idFieldName: "postId",
+      resolverName: "post",
+      collectionName: "Posts",
+      type: "Post",
+    }),
+    canRead: ['guests'],
+    canCreate: ['members'],
+  },
+  subforumLastVisitedAt: {
+    type: Date,
+    optional: true,
+    nullable: true,
+    hidden: true
+  },
+};
+
+export const UserTagRels: UserTagRelsCollection = createCollection({
+  collectionName: 'UserTagRels',
+  typeName: 'UserTagRel',
+  schema,
+  // We don't need graphql access currently, but when we do you'll want to add these back in:
```

Comment:
I deliberately made it so this collection isn't directly accessible via graphql to simplify this PR, but we may need to add graphql access at some point in the future